src/components/steps/step-sheet/CodeTab.tsx

```diff
@@ -15,9 +17,49 @@ export function StepCodeTab({ stepId }: Props) {
 
 	if (isPending) return <Skeleton className="h-[300px] w-full" />;
 
+	const traceback = data?.metadata?.exception_info?.traceback;
+
+	function goToError() {
+		const errorLineElement = document.querySelector(".error-highlight-line");
+		if (errorLineElement) {
+			errorLineElement.scrollIntoView({ behavior: "smooth" });
+		}
+	}
+
 	return (
-		<CollapsibleCard initialOpen title="Code">
-			<Codesnippet fullWidth highlightCode wrap code={data?.metadata?.source_code || ""} />
-		</CollapsibleCard>
+		<div className="flex flex-col gap-5">
+			{traceback && (
+				<CollapsibleCard
+					initialOpen
+					title={
+						<>
+							<AlertCircle className="h-5 w-5 fill-red-500" /> Error
+						</>
+					}
+					className="border border-error-200"
```

Comment:
<img width="1120" height="188" alt="grafik" src="https://github.com/user-attachments/assets/8205f06e-ab74-46c7-941b-76d34df15119" />
<img width="1033" height="161" alt="grafik" src="https://github.com/user-attachments/assets/929b24a3-5b47-4232-b527-f237368c3cb2" />

The traceback doesnt align with the design. I think the design doesnt use a collapsible here, and also the colors (background, icon, etc...) dont align